help_to_heat/frontdoor/schemas.py

```diff
@@ -86,6 +92,9 @@
 household_pages = {
     "country": ("country",),
     "supplier": ("supplier",),
+    "park-home": ("park_home",),
+    "park-home-main-residence": ("park_home_main_residence",),
+    "park-home-application-closed": ("park_home_application_closed",),
```

Comment:
Still got some underscores kicking about. These ones are page names, not data fields 🙂

Reply:
Sorry I got a bit confused now :( I thought they need underscores to match those in the `summary_map`, so that we could find the corresponding summary line question? 🤔

Reply:
If I understanding is correct it should be like this as application-closed pages won't have matching summary lines 🤔 Everything else is working as before
![image](https://github.com/UKGovernmentBEIS/help-to-heat-GBIS/assets/104062694/b719aaa1-d59b-447c-ac16-d592b4c269a0)

Reply:
Suspect you're right, just annoyed/confused by the warning pages 😓 

Fine as-is, ty

Reply:
Cool! Yeah it is ready confusing 😅